libs/blocks/src/lib/cta/cta-left/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Heading, Section, Text, qtMerge } from '@deriv/quill-design';
+
+export type CTABlockProps = {
+  content?: React.ReactNode;
+  children?: React.ReactNode;
+  title: React.ReactNode | string;
+  description?: React.ReactNode | string;
+  className?: string;
+  variant?: 'content-left' | 'content-right';
+};
+export const CTALeft = ({
+  content,
+  children,
+  title,
+  description,
+  variant = 'content-left',
+}: CTABlockProps) => {
+  return (
+    <Section className=" py-general-4xl">
+      <div
+        className={qtMerge(
+          variant === 'content-left' ? 'lg:flex-row' : 'lg:flex-row-reverse ',
+          ' relative isolate flex h-full min-h-[496px] justify-end gap-gap-lg  max-lg:flex-col md:justify-center lg:mx-auto  lg:max-w-[1232px] ',
+        )}
+      >
+        {content && (
+          <>
+            <div className="absolute inset-50 z-10 bg-cta-gradient lg:bg-none"></div>
+            <div className="-z-10 h-full w-full overflow-hidden  object-cover object-top max-lg:absolute lg:relative ">
+              {content}
+            </div>
+          </>
+        )}
+        <div
+          className={qtMerge(
+            variant === 'content-left'
+              ? 'lg:ps-general-2xl '
+              : 'lg:pe-general-2xl ',
+            'flex w-full flex-col justify-center gap-gap-3xl px-general-lg pb-general-2xl md:justify-center lg:px-general-none lg:pb-general-none',
+          )}
+        >
+          <div className="flex w-full flex-col gap-gap-lg">
+            {title && (
+              <>
+                <Heading.H2 className="hidden lg:block">{title}</Heading.H2>
+                <Heading.H2 className="z-20 block text-solid-slate-50 max-sm:w-full sm:w-[350px] lg:hidden lg:w-full">
+                  {title}
+                </Heading.H2>
```

Comment:
Why two heading? and why `max-sm`?

Reply:
Hi @meenakshi-deriv can you please help to check @hasan-deriv's question ?

Reply:
```suggestion
                <Heading.H2 className="hidden lg:block">{title}</Heading.H2>
                <Heading.H2 className="z-20 block text-solid-slate-50 w-full sm:w-[350px] lg:hidden">
                  {title}
                </Heading.H2>
```

Reply:
hi @hasan-deriv this is because some tailwind classes were not applying on deriv-hk-gold. we tried many ways but only this approach fixed the issue.